cupy/_core/include/cupy/complex/arithmetic.h

```diff
@@ -18,7 +18,7 @@
 #pragma once
 
 #include <cupy/complex/complex.h>
-#include <cupy/complex/math_private.h>
+#include <cfloat>
```

Comment:
Let me try including cfloat as the latest CUDA includes it in the corresponding thrust header instead of `math_private.h`.

Reply:
Okay, thanks!